securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -254,6 +270,14 @@ Example polyfill libraries:
 * HTML5 History - https://github.com/devote/HTML5-History-API
 * Promise - https://github.com/stefanpenner/es6-promise
 
+==== A note about `promiseType`
+
+Throughout the examples in the documentation you will find `promiseType: 'native'` specified wherever Keycloak is initialized.
```

Comment:
Should Keycloak be {project_name} throughout this content?
Otherwise, looks good.

Reply:
Good catch, I've added it in the places where it seemed appropriate. It also seems like there are a lot of instances where this is not consistently adhered to as well, but that is outside of the scope of this PR.